boards/FSM/Src/App/App_Brake.c

```diff
@@ -0,0 +1,70 @@
+#include <stdlib.h>
```

Comment:
@garethellis0 @thekenu Can you guys check to see if this is along the lines of what you were looking for b4 I continue? I added moved the `IsBrakeActuated` to `Io_Brake`, and used `App_Brake` to wrap around the Io fxns